Sources/MapboxMaps/Foundation/Camera/CameraView.swift

```diff
@@ -166,83 +42,42 @@ internal class CameraView: UIView {
                              pitch: localPitch)
     }
 
-    private unowned var delegate: CameraViewDelegate!
-
-    init(delegate: CameraViewDelegate, edgeInsets: UIEdgeInsets = .zero) {
-        self.delegate = delegate
+    init() {
         super.init(frame: .zero)
-
         self.isHidden = true
         self.isUserInteractionEnabled = false
-
-        // Sync default values from MBXMap
-        setFromValuesWithMapView()
     }
 
     internal required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
-    private func setFromValuesWithMapView() {
-        zoom = delegate.zoom
-        bearing = CGFloat(delegate.bearing)
-        pitch = delegate.pitch
-        padding = delegate.padding
-        centerCoordinate = delegate.centerCoordinate
-    }
-
-    internal func update() {
-
-        // Retrieve currently rendered camera
-        let currentCamera = delegate.camera
-
-        // Get the latest interpolated values of the camera properties (if they exist)
-        let targetCamera = localCamera.wrap()
-
-        // Apply targetCamera options only if they are different from currentCamera options
-        if currentCamera != targetCamera {
-
-            // Diff the targetCamera with the currentCamera and apply diffed camera properties to map
-            let diffedCamera = CameraOptions()
-
-            if targetCamera.zoom != currentCamera.zoom, let targetZoom = targetCamera.zoom, !targetZoom.isNaN {
-                diffedCamera.zoom = targetCamera.zoom
-            }
-
-            if targetCamera.bearing != currentCamera.bearing, let targetBearing = targetCamera.bearing, !targetBearing.isNaN {
-                diffedCamera.bearing = targetCamera.bearing
-            }
-
-            if targetCamera.pitch != currentCamera.pitch, let targetPitch = targetCamera.pitch, !targetPitch.isNaN {
-                diffedCamera.pitch = targetCamera.pitch
-            }
-
-            if targetCamera.center != currentCamera.center, let targetCenter = targetCamera.center, !targetCenter.latitude.isNaN, !targetCenter.longitude.isNaN {
-                diffedCamera.center = targetCamera.center
-            }
-
-            if targetCamera.anchor != currentCamera.anchor {
-                diffedCamera.anchor = targetCamera.anchor
-            }
+    internal func syncLayer(to cameraOptions: CameraOptions) {
+        if let zoom = cameraOptions.zoom {
+            layer.opacity = Float(zoom)
+        }
 
-            if targetCamera.padding != currentCamera.padding {
-                diffedCamera.padding = targetCamera.padding
-            }
+        if let bearing = cameraOptions.bearing {
+            layer.cornerRadius = CGFloat(bearing)
+        }
```

Comment:
Could you explain this with some documentation. This feels weird setting the `cornerRadius` to a bearing. What is the purpose of this?

Reply:
This is the same thing we [were doing earlier](https://github.com/mapbox/mapbox-maps-ios/blob/7c6333e4967333b11c9a80070af9e6cf9dd6d60c/Sources/MapboxMaps/Foundation/Camera/CameraView.swift#L75-L83).. It's just been refactored here.

Reply:
I guess I haven't actively looked at that previous code. But what is the correlation between `cornerRadius` and `bearing`?

Reply:
lets 1:1